roles/st2repos/tasks/redhat.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Define RH Version
+  set_fact:
+      rh_ver: "{{ ansible_distribution_version|int|abs }}"
+  # TODO(mierdin): Remove when https://github.com/StackStorm/ansible-st2/issues/89 is addressed
+
+- name: Add ST2 Repo
+  become: yes
+  yum_repository:
+    name: "StackStorm_{{ st2_pkg_repo }}"
+    description: "StackStorm_{{ st2_pkg_repo }}"
+    file: st2
```

Comment:
Naming should follow the PackageCloud script install logic where possible.
I see `/etc/yum.repos.d/StackStorm_stable.repo` for my installation.